changelog.txt

```diff
@@ -2,6 +2,8 @@
 
 = 6.9.0 - xxxx-xx-xx =
 * Fix - Resolved an issue that could cause an empty error notice to appear on the My Account > Payment methods page when a customer attempted to delete a token used by subscriptions.
+* Fix - Use block theme styles for the 'Add to Cart' button on subscription product pages.
```

Comment:
I added this changelog for a recent PR i merged: https://github.com/Automattic/woocommerce-subscriptions-core/pull/574

Reply:
Thanks when will this release come out?